glustercli/cmd/glustershd.go

```diff
@@ -29,6 +38,13 @@ func init() {
 	selfHealCmd.AddCommand(selfHealIndexCmd)
 	selfHealCmd.AddCommand(selfHealFullCmd)
 
+	selfHealSplitBrainCmd.Flags().BoolVar(&flagSplitBrainBiggerFile, "bigger-file", false, "Use bigger-file to resolve split-brain")
+	selfHealSplitBrainCmd.Flags().BoolVar(&flagSplitBrainLatestMtime, "latest-mtime", false, "Use latest-mtime to resolve split-brain")
+	selfHealSplitBrainCmd.Flags().BoolVar(&flagSplitBrainSourceBrick, "source-brick", false, "Use a brick as source to resolve split-brain")
+	selfHealSplitBrainCmd.Flags().StringVar(&flagFileName, "file", "", "Specify filename that is in split-brain")
+	selfHealSplitBrainCmd.Flags().StringVar(&flagBrickHostAndPath, "brick", "", "Specify hostname and brick path in format <hostname:brickpath>")
```

Comment:
The CLI structure in glusterd is somewhat similar. I have added flagSplitBrainSourceBrick just to make sure which type of split resolution use wants and then an additional flag flagBrickHostAndPath to supply the source-brick.

Reply:
Can we receive the brick host and path in the source brick itself, because IMO this one flag is enough to get the necessary arguments and also say what type of splitbrain resolution it is.